src/ripple/protocol/impl/LedgerFormats.cpp

```diff
@@ -101,13 +101,11 @@ LedgerFormats::LedgerFormats ()
             ;
 
     add ("Amendments", ltAMENDMENTS)
-            << SOElement (sfLedgerSequence,      SOE_OPTIONAL)
```

Comment:
So should I add myself and @nbougalis as reviewers and mark the pull request passed?